projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/repository/jpa/SampleCodeConverter.java

```diff
@@ -0,0 +1,27 @@
+package life.qbic.projectmanagement.domain.project.repository.jpa;
+
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+import life.qbic.projectmanagement.domain.project.sample.SampleCode;
+
+/**
+ * <b>Converts {@link SampleCode} into a String and vice versa></b>
+ *
+ * <p>Converts the String value stored in the database to an
+ * {@link SampleCode}. Additionally converts the {@link SampleCode} to a string value to be stored
+ * in the database.
+ * </p>
+ */
+@Converter(autoApply = true)
```

Comment:
what does the option value "autoApply" do?

Reply:
From the docs: 
If the autoApply element is specified as true, the persistence provider must automatically apply the converter to all mapped attributes of the specified target type for all entities in the persistence unit except forattributes for which conversion is overridden. 
AFAIK we always want the conversion to be applied to the sampleCode wthin our domain/persistence layer